src/processes/helper/sendTopicAutoMessage.js

```diff
@@ -27,7 +27,10 @@ const followTopicServiceQueue = async (userId, topicId, communityMessageFormatId
     delay: diffTime
   };
   try {
-    await followTopicQueue.add(data, options);
+    const queueRegistered = await followTopicQueue.add(data, options);
+    console.log(
+      `::: followTopicServiceQueue for user_id ${userId} and communityMessageFormatId ${communityMessageFormatId} :::  ${queueRegistered.id}`
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logging mechanism added here is a good way to track the queue registration ID. However, it's not recommended to use `console.log` for production code as it can slow down your application and expose sensitive information. Consider using a dedicated logging library like Winston or Bunyan which provides more control over log management.

Also, the error handling could be improved. Currently, it just logs the error but does not rethrow it or handle it in any other way. This might lead to silent failures where an error occurs but the program continues to run as if nothing happened.

Here's how you could improve this:

```diff
   try {
     const queueRegistered = await followTopicQueue.add(data, options);
-    console.log(
+    logger.info(
       `::: followTopicServiceQueue for user_id ${userId} and communityMessageFormatId ${communityMessageFormatId} :::  ${queueRegistered.id}`
     );
   } catch (e) {
-    console.log('error', e);
+    logger.error('Error while adding to followTopicQueue', e);
+    throw e;
   }
```
In this snippet, `logger` is assumed to be an instance of a logging library. The error is logged with a descriptive message and then rethrown so that it can be handled by the caller.


<!-- This is an auto-generated comment by OSS CodeRabbit -->